table/pit-cs.go

```diff
@@ -133,7 +133,7 @@ func (bpe *basePitEntry) InsertInRecord(interest *ndn.Interest, face uint64, inc
 		record.ExpirationTime = time.Now().Add(interest.Lifetime())
 		record.PitToken = incomingPitToken
 		bpe.inRecords[face] = record
-		return record, len(bpe.inRecords) > 1
+		return record, false
```

Comment:
See https://github.com/danningyu/YaNFD/pull/2/files#r837020735.

Should've been changed there but the change got lost in the processing of rebasing in git.